docs/concepts/crds/backupconfiguration/index.md

```diff
@@ -5,293 +5,342 @@ menu:
+
+#### HookInfo
+HookInfo specifies the information about the backup hooks. It contains the following fields:
+
+| Field             | Usage                                                                                                                                                                                                                                                                                                                                                                                                         |
+|-------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `name`            | specifies a name for the hook                                                                                                                                                                                                                                                                                                                                                                                 |
+| `hookTemplate`    | points to a HookTemplate CR that will be used to execute the hook. You can refer to a HookTemplate from other namespaces as long as your current namespace is allowed by the `usagePolicy` in the respective HookTemplate.                                                                                                                                                                                    |
+| `params`          | specifies parameters for the hook. You must provide the parameter in the HookTemplates desired structure.                                                                                                                                                                                                                                                                                                     |
+| `maxRetry`        | MaxRetry specifies how many times Stash should retry the hook execution in case of failure.   The default value of this field is 0 which means no retry.                                                                                                                                                                                                                                                      |
```

Comment:
```suggestion
| `maxRetry`        | MaxRetry specifies how many times KubeStash should retry the hook execution in case of failure.   The default value of this field is 0 which means no retry.                                                                                                                                                                                                                                                      |
```